lib/project_types/rails/commands/create.rb

```diff
@@ -28,7 +28,7 @@ def call(args, _name)
         return @ctx.puts(self.class.help) if form.nil?
 
         @ctx.abort(@ctx.message('rails.create.error.invalid_ruby_version')) unless
-          Ruby.version(@ctx).satisfies?('~>2.4')
+          Ruby.version(@ctx).satisfies?('~>2.5')
```

Comment:
This will be run with the locked-in CLI version, right? In that case, do we want to bump the Ruby version requirement for the CLI to 2.5 as well?